communications.md

```diff
@@ -1,89 +1,49 @@
 
-Opt-out: Leave channels that you are no longer interested in or obligated to.
+## Choosing appropriate tools
 
-Asynchronous Communication by Default: Whenever you want to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and their productivity and the fact that they may be in the "zone" and should not be interrupted. 
+Our communication tools encompasses both formal and informal communications so we may want to consider the following factors:
+* Your audience (Is this important to the Locus team or Atom team?)
+* The size of the audience (Is this relevant to one person or a whole team?)
+* Content of message and reusability (Will this message be useful to someone else in the future?)
 
-When necessary, elevate your message's attention-seeking priority in such order:
+In general, if it’s applicable to more than one person, share in the relevant Slack channel or Asana project so it can be searchable and found in an accessible manner. For example, asking questions in a Slack channel allow responses to those questions to become part of our knowledge base. Only exception is sharing common credentials or secrets - we have a well-controlled private channel for this.
 
-1. Send a well-written message in a relevant Slack channel
-2. If this needs a certain person's or group's attention, ping them right below or start a thread and ping them there.
-3. In a rare "meltdown" level of urgency, yell out in any synchronous manner and get people's attention.
+Each person communicates and receives information differently so if something requires attention, feel free to verbally ask colleagues or check their Slack status for how they would be like to be communicated. e.g. ping on Slack versus asking in person.
 
-Documentation - Consolidation: Asynchronous communication can become chaotic so it should be every participant's, at least the initiator's obligation to organize and summarize afterward and write it in well-formatted documentation as a part of our internal knowledge base. At a bare minimum, copy the transcript of relevant discussions into a place for future reference or refinement to be retained in the knowledge base.
+When there is doubt, use Slack.
 
-Documentation - Transparency: Don't be embarrassed about asking "silly" questions. Try to always write your messages in a publically accessible (or at least a team accessible) channel. This way, the information is preserved and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
+## Meetings
 
-A few exceptions:
+If a meeting is necessary, we have meeting spaces that can be pre-booked and used appropriately. Please be mindful of the duration time and goal for the meeting as we presently have limited private space for meetings and others might be waiting in line. If you’d like to get tips on how to ensure a productive meeting, click [here](https://github.com/EQWorks/common/productive-meeting.md).
```

Comment:
Shall we mention the meeting agenda part? Personally, I think to have a meeting agenda before the meeting will help people understand whether they should attend this meeting or what should prepare for this meeting to save some time.

Reply:
great point. I was initially going to put that in productive meeting link but that makes sense. will add that.